.github/workflows/ci.yml

```diff
@@ -161,6 +163,12 @@ jobs:
       - name: Set up MATLAB
         uses: matlab-actions/setup-matlab@v1.2.3
 
+      - name: Install HDF5 libraries for Ubuntu
+        if: ${{ contains(matrix.os, 'ubuntu') }}
+        run: |
+          sudo apt-get update
+          sudo apt-get install libhdf5-dev
```

Comment:
Seems to work, but we may want to statically link when compiling. (Actually, I thought we were doing this...?)